FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/PostHelpFeedbackResponse.kt

```diff
@@ -52,7 +52,7 @@ data class PostHelpFeedbackResponse constructor(
   override fun writeToParcel(dest: Parcel, flags: Int) =
     with(dest) {
       writeParcelable(apiError, 0)
-      writeInt((if (isSent) 1 else 0))
+      writeInt((if (isSent == true) 1 else 0))
```

Comment:
shouldn't we get into account that `isSent` can be null and serialize something different for `null`?

Reply:
Good one. I will create a separate issue for that.